brd-ios/breadwallet/src/Extensions/UserDefaults+Additions.swift

```diff
@@ -51,7 +51,7 @@ private let appLaunchesAtLastNotificationDeferralKey = "appLaunchesAtLastNotific
 private let deviceIdKey = "BR_DEVICE_ID"
 private let savedChartHistoryPeriodKey = "savedHistoryPeriodKey"
 private let balanceKey = "balanceKey"
-private let sessionKey = "sessionKey"
+private let walletToken = "sessionKey"
```

Comment:
I suggest changing the string value as well - but migrating this would require you to copy value of sessionKey over

Reply:
I didn't want to mess with it